lms/templates/instructor/instructor_dashboard_2/send_email.html

```diff
@@ -9,38 +9,52 @@
   <div class="request-response msg msg-confirm copy" id="request-response"></div>
   <ul class="list-fields">
     <li class="field">
-      <label for="id_to">${_("Send to:")}</label><br/>
-      <select id="id_to" name="send_to">
-        <option value="myself">${_("Myself")}</option>
-        %if to_option == "staff":
-        <option value="staff" selected="selected">${_("Staff and admins")}</option>
-        %else:
-        <option value="staff">${_("Staff and admins")}</option>
-          %endif
-          %if to_option == "all":
-            <option value="all" selected="selected">${_("All (students, staff, and admins)")}</option>
-          %else:
-      <option value="all">${_("All (students, staff, and admins)")}</option>
-      %endif
-      </select>
+        ${_("Send to:")}
+        <ul role="group" aria-label="${_('Send to:')}">
```

Comment:
I'm not clear about having an aria-label that repeats the text above it. I'd check with someone on accessibility to see what they recommend.